### PR TITLE
Added Windows Install Markdown File Separate of INSTALL.md

### PR DESCRIPTION
**Pull request by jonathanzhang53**
Added Windows Install File. Took information from INSTALL.md file and added changes. #1288 includes INSTALL.md changes, but those are not included in this PR.

### CODE REVIEW

**Comment by alpinus4** [comment]
For the clarity, please rename the file to WINDOWS_INSTALL.md

**Comment by KoniKodes** [comment]
> **Q. I am getting a `Check that mysqld is running` error when opening up EHR!**

Should this be "mysql"?

**Comment by jonathanzhang53** [comment]
> > **Q. I am getting a `Check that mysqld is running` error when opening up EHR!**
> 
> Should this be "mysql"?

"mysqld" is MySQL server daemon program which runs quietly in background on your computer system. Invoking "mysqld" will start the MySQL server on your system. Terminating "mysqld" will shutdown the MySQL server. I remember getting the error when I had fetched the upstream and it changed my database credentials.

@KoniKodes

**Comment by jonathanzhang53** [comment]
This can only really be tested after 7.1 or 7.2 has been made compatible with ehr anyway, but assuming the version is compatible, these instructions should be good, but not merging is a good idea for now. @robbyoconnor

**Comment by KoniKodes** [comment]
> > > **Q. I am getting a `Check that mysqld is running` error when opening up EHR!**
> > 
> > 
> > Should this be "mysql"?
> 
> "mysqld" is MySQL server daemon program which runs quietly in background on your computer system. Invoking "mysqld" will start the MySQL server on your system. Terminating "mysqld" will shutdown the MySQL server. I remember getting the error when I had fetched the upstream and it changed my database credentials.
> 
> @KoniKodes
I forgot about deamons. That makes sense then.

**Comment by jonathanzhang53** [comment]
@KoniKodes did you get a chance to read over the rest of it? How does it look?

**Comment by KoniKodes** [comment]
@jonathanzhang53 
This is so much easier to read and understand.
Please submit it, thank you.

File: WINDOWS_INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+```
+$pass    = 'libreehr';
+```
+
+Restart Apache Web Server and MySQL.
+
+**Q. How do I get back to the Login Page once I close out?**
+
+Use the login path with localhost. You may also check your browser history. It is recommended to bookmark this site for easy access. Example: [`http://localhost/lh-ehr/interface/login/login.php?site=default`](http://localhost/lh-ehr/interface/login/login.php?site=default)
```
Comment (muarachmann) [inline WINDOWS_INSTALL.md]: I think pointing to `http://localhost/lh-ehr` does the same thing. Takes you to login page if you have the software installed

Comment (jonathanzhang53) [reply WINDOWS_INSTALL.md]: It works. I edited the file.

**Comment by tmccormi** [comment]
Good work.

**Pull request merged**